### PR TITLE
chore: remove fio-spdk from release

### PR DESCRIPTION
**Pull request by tiagolobocastro**
This was added by accident.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -167,6 +167,9 @@ pipeline {
           }
           agent { label 'nixos-mayastor' }
           steps {
+            cleanWs()
```
Comment (mtzaurus) [inline Jenkinsfile:170]: Is this the same as in 1221 ?

**Review by Ab-hishek — APPROVED** [review]

**Review by mtzaurus — APPROVED** [review]

**Comment by mtzaurus** [comment]
If this PR in fact extends 1221, you can merge just this one as it covers both changes.

**Comment by tiagolobocastro** [comment]
> If this PR in fact extends 1221, you can merge just this one as it covers both changes.

This is for release and 1221 is for develop

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**